geospaas_harvesting/harvest.py

```diff
@@ -12,6 +12,7 @@
 import time
 from datetime import datetime
 import django
+import django.db
```

Comment:
is that line working alone (I mean `django.db.connection.close()`) without importing django.db?

Reply:
It seems to work in that case, but it's a good practice to always import the sub-modules you use because by default, sub-modules are not imported automatically when you import a package.